nautobot_golden_config/datasources.py

```diff
@@ -125,11 +122,24 @@ def get_id_kwargs(gc_config_item_dict, id_keys, job_result):
         if actual_attr_name in fk_class_mapping:
             if "network_driver" in yaml_attr_name:
                 field_name = "network_driver"
+            elif "platform_name" in yaml_attr_name:
```

Comment:
Does it matter which is first if we want platform_name to be preferred?

Reply:
this piece should be determining that.

```python
    if "platform_name" in gc_config_item_dict.keys() and "platform_network_driver" in gc_config_item_dict.keys():
        gc_config_item_dict.pop("platform_network_driver")
```